services/distribution/src/main/resources/application.yaml

```diff
@@ -16,6 +16,8 @@ services:
   distribution:
     # The name of the distribution output file.
     output-file-name: index
+    # The name of the distribution output file v2.
+    output-file-name-v2: index-v2
```

Comment:
`index-v2` ??? where is this used? the path of the configs should be a different one... I don't think we should have any file, that has index-v2 as file name, right?

Reply:
We need this file because this is the way the directory structure of the v1, v2 config is developed.